lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
To properly reflect the intent, we should have then `if (!rules || rules.length === 0) {` right?

I am afraid ` === void 0` has no real signification for a user.